### DISCUSSION TITLE
eventContent function return undefined

### DISCUSSION THREAD

**Discussion by lukas-frey** [vkurko/calendar#306] [Q&A] [answered]
Hi, would it be possible to render the default event, when the function in `eventContent` returns `undefined`?

In my case, I have a function that conditionally returns different HTML and sometimes I need it to fallback to the default view. But `eventContent` right now only renders the default view when it is directly set to `undefined`.

Thanks :)

**Comment by vkurko** [comment, marked as answer]
Yes, I think this will be a very useful feature. I will implement it in a future version.

**Reply by lukas-frey** [reply]
Awesome, thanks!

**Reply by vkurko** [reply]
The ability to apply default rendering in the `eventContent` callback should be available in Event Calendar v3.4.0. Please check.

**Reply by lukas-frey** [reply]
Works perfectly, thank you!
